_activities/idds.md

```diff
@@ -0,0 +1,25 @@
+---
+title: intelligent Data Delivery Service
+author: Wen Guan
+layout: plain
+---
+
+## Introduction
+
+iDDS(intelligent Data Delivery Service) is an intelligent Data Delivery Service. It's an ATLAS/IRIS-HEP joint project. It’s designed to intelligently transform and deliver the needed data to the processing workflow in a fine-grained approach for High Energy Physics workloads. It will not only reduce the need for replicas, but also enable benefits such as decreasing the time period of caching transient data, transforming expensive replicas to cheaper format data at remote sites and only cache cheaper new data for processing. iDDS will also work to orchestrate the WorkFlow Management System (WFMS) and the Distributed Data Management (DDM) systems to trigger them to process the data as soon as possible. In addition, iDDS will have intelligent algorithms to adjust the lifetime of cache, the format transformation and the delivery destination. The iDDS will increase the efficiency of data usage, reduce storage usage for processing and speed up the processing workflow.
```

Comment:
The first sentence here seems to have an extra *intelligent Data Delivery Service* in it. I think would be better as

> iDDS is an *intelligent Data Delivery Service*

I would also avoid "it's" - the page should be a little more formal... so "it is" please!

The 3rd sentence is quite long and I think is clearer if you punctuate it like this (and add a few articles, as below):

> It will not only reduce the need for replicas, but also enable benefits such as: decreasing the time period of caching transient data; transforming expensive replicas to *a* cheaper format data at remote sites; and only cache *the* cheaper new data for processing

I think it will be easier on the eye if you put a paragraph break after that sentence, at the start of "iDDS will also work to orchestrate the WorkFlow...".

And "lifetime of cache" should be "lifetime of *the* cache".

Reply:
I updated to fix them.